README.md

```diff
@@ -128,18 +128,20 @@ must [supply credentials](#authentication) and a project ID if running this snip
 import com.google.gcloud.bigquery.BigQuery;
 import com.google.gcloud.bigquery.BigQueryOptions;
 import com.google.gcloud.bigquery.Field;
+import com.google.gcloud.bigquery.Job;
 import com.google.gcloud.bigquery.JobStatus;
 import com.google.gcloud.bigquery.JobInfo;
 import com.google.gcloud.bigquery.LoadJobConfiguration;
 import com.google.gcloud.bigquery.Schema;
 import com.google.gcloud.bigquery.StandardTableDefinition;
+import com.google.gcloud.bigquery.Table;
 import com.google.gcloud.bigquery.TableId;
 import com.google.gcloud.bigquery.TableInfo;
 
 BigQuery bigquery = BigQueryOptions.defaultInstance().service();
 TableId tableId = TableId.of("dataset", "table");
-TableInfo info = bigquery.getTable(tableId);
-if (info == null) {
+Table table = bigquery.getTable(tableId);
+if (table == null) {
```

Comment:
I wonder if, as an example, the 'else' part is needed (vs, create if needed and upload data...)